hardware/firmware/Android.mk

```diff
@@ -1,14 +0,0 @@
-# Mount points
-
-LOCAL_PATH := $(call my-dir)
-
-include $(CLEAR_VARS)
-LOCAL_MODULE := firmware_folders
-LOCAL_MODULE_PATH := $(TARGET_OUT_VENDOR)
-
-LOCAL_POST_INSTALL_CMD := mkdir -p \
-    $(TARGET_OUT_VENDOR)/dsp \
-    $(TARGET_OUT_VENDOR)/firmware_mnt \
-    $(TARGET_OUT_VENDOR)/bt_firmware
```

Comment:
This commit makes no sense to me. These are the _folders in `/vendor`_ where the partitions will be mounted. The `BOARD_ROOT_EXTRA_SYMLINKS` in CommonConfig.mk _link to these folders_, not the other way around.

Reply:
Dropped